docs/interchange.rst

```diff
@@ -55,7 +56,9 @@ sample
     Those nodes in the tree that we have obtained data from.  These are
     distinguished from other nodes by the fact that a tree sequence *must*
     describe the genealogical history of all samples at every point on the
-    genome. (See the :ref:`node table definitions <sec_node_table_definition>`
+    genome. Certain methods depend on these: for instance,
+    :meth:`SparseTree.roots` only return roots ancestral to at least one sample.
+    (See the :ref:`node table definitions <sec_node_table_definition>`
```

Comment:
Perhaps there should be a list of things that treat the samples as the tips in this way?